server/opts.go

```diff
@@ -2482,6 +2486,9 @@ func setBaselineOptions(opts *Options) {
 	if opts.ReconnectErrorReports == 0 {
 		opts.ReconnectErrorReports = DEFAULT_RECONNECT_ERROR_REPORTS
 	}
+	if opts.MaxTracedMessagePrintableLen == 0 {
```

Comment:
Should we have something to designate no limit? -1 maybe?

Reply:
Good catch. Will change default value for `MaxTracedMessagePrintableLen` to `-1` to designate no limit.